src/containers/SubjectViewer.jsx

```diff
@@ -25,9 +33,21 @@ class SubjectViewer extends React.Component {
     this.onImageLoad = this.onImageLoad.bind(this);
     this.onImageError = this.onImageError.bind(this);
     this.getBoundingBox = this.getBoundingBox.bind(this);
+    this.onMouseDown = this.onMouseDown.bind(this);
```

Comment:
👍 for the implementation as is, but someone please remind me...


1. I need to check if events can use React's onMouseDown and onMouseUp. I remember losing some data in the synthetic React events, but I should double check.
2. Also check if React's onMouseDown/Up/etc events also captures **touch input**
3. If either 1 or 2 doesn't pan out, I need to implement the touch input code. (Video games to the rescue!)

Reply:
Re: item 1, Wait, what am I talking about? The `<svg>` element here DOES use proper React onMouseDown and onMouseUp. i.e. `<svg onMouseDown={this.onMouseDown}>`, now `this.svg.addEventHandler('onmousedown', this.onMouseDown)`

Sorry, it turns out my brain was on the fritz last night. OK, that just means I need to figure out if [React's Touch events.](https://reactjs.org/docs/events.html#touch-events)